plugins/python-build/share/python-build/pypy-2.0-src

```diff
@@ -1,4 +1,4 @@
 #require_gcc
 prefer_openssl11
 install_package "openssl-1.1.0j" "https://www.openssl.org/source/old/1.1.0/openssl-1.1.0j.tar.gz#31bec6c203ce1a8e93d5994f4ed304c63ccf07676118b6634edded12ad1b3246" mac_openssl --if has_broken_mac_openssl
-install_package "pypy-2.0-src" "https://bitbucket.org/pypy/pypy/downloads/pypy-2.0-src.tar.bz2#d92dfd418beac915d3efc28f8a2090f3c13a89ec653419deff3d7112c5c111f3" "pypy_builder" verify_py27 ensurepip
+  install_package "pypy-2.0-src" "https://downloads.python.org/pypy/pypy-2.0-src.tar.bz2#d92dfd418beac915d3efc28f8a2090f3c13a89ec653419deff3d7112c5c111f3" "pypy_builder" verify_py27 ensurepip
```

Comment:
I was wondering if those whitespace changes on the beginning of the line were intentional.

Reply:
It happened because awk discards them. I agree, they have to be fixed.

But apart from that, I would be grateful if someone can verify the installation works and the checksums (?) are correct.